src/subscriber/subscriberV2.ts

```diff
@@ -62,6 +64,19 @@ export default class SubscriberV2 extends Subscriber {
             },
           };
 
+          constructor(...args: any[]) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
          constructor(...args: unknown[]) {
```